tools/ncf_rudder.py

```diff
@@ -271,7 +273,7 @@ def generate_rudder_reporting(technique):
     method_name = method_call['method_name']
     generic_method = generic_methods[method_name]
 
-    key_value = method_call["args"][generic_method["class_parameter_id"]-1]
+    key_value = method_call["args"][generic_method["class_parameter_id"]-1].replace("\\'", "\'").replace('\\"', '\"')
```

Comment:
i guess this is not relevant anymore; we changed a lot of things around that (notably, we don't use bash anymore to generate the reporting file)